libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/camera/SimpleCamera.java

```diff
@@ -63,15 +50,11 @@ private void buildRouteCoordinatesFromRouteData(RouteInformation routeInformatio
   }
 
   private void setupLineStringAndBearing(DirectionsRoute route) {
-    if (initialRoute != null && route.equals(initialRoute)) {
+    if (route.equals(initialRoute)) {
       return; //no need to recalculate these values
     }
     initialRoute = route;
     routeCoordinates = generateRouteCoordinates(route);
```

Comment:
@LukasPaczos can we double check that these `routeCoordinates` are still being used?  I believe they are (for the route overview `LatLngBounds`) but just want to double check - thanks!

Reply:
Looks like they are being used for the overview as you mentioned :)